spec/ewb/IEC61968/AssetInfo/WireMaterialKind.yaml

```diff
@@ -20,8 +20,8 @@ attributes:
 - name: other
   description: Other wire material.
 - name: UNKNOWN
-  description: Default. Evolve extension.
+  description: Default. EWB extension.
```

Comment:
Renaming Evolve to EWB - I am aware that new things are marked as ZBEX, but I have a separate ticket to actually introduce proper extension support so that the markdown is colored/highlighted nicely

Reply:
Ticket here https://app.clickup.com/t/6929263/DEV-899